src/index.html

```diff
@@ -13,5 +13,166 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    onsubmit="onSubmit()">
+
+      <!-- personal -->
+      <fieldset class="fieldset">
+        <legend> Personal information</legend>
+        <label class="form-field" for="surname">
+          Surname:
+          <input type="text"
+          autocomplete="false"
+          name="surname"
+          id="surname"
+          minlength="3"
+          maxlength="10">
+        </label>
+
+        <label class="form-field" for="name">
+          Name:
+          <input type="text"
+          autocomplete="false"
+          name="name"
+          id="name">
+        </label>
+
+        <label class="form-field" for="age">
+          How old are you?
+          <input type="number"
+          value="12"
+          name="age"
+          max="100"
+          min="12"
+          id="age">
+        </label>
+
+        <label class="form-field" for="birth">
+          Full date of birth:
+          <input type="date"
+          name="birthday"
+          id="birth">
+        </label>
+
+        <label class="form-field" for="terms">
+          I accept the term of agreement
+          <input type="checkbox"
+          name="agreement"
+          id="terms">
+        </label>
+      </fieldset>
+
+      <!-- register  -->
+      <fieldset class="fieldset">
+        <legend> Registration </legend>
+        <label class="form-field" for="email">
+          E-mail:
+          <input type="email"
+          name="email"
+          placeholder="email@example.com"
+          id="email"
+          required>
+        </label>
+
+        <label class="form-field" for="password">
+          Password:
+          <input type="password"
+          name="email"
+          id="password">
+        </label>
+      </fieldset>
+
+      <!-- interesting facts -->
+      <fieldset class="fieldset">
+        <legend> An interesting fact about you! </legend>
+        <div class="form-field">
+          Do you love cats?
+
+          <label for="cats-y">
```

Comment:
```suggestion
          <label for="cats-yes">
```
Let's use more meaningful names